castai/resource_workload_scaling_policy.go

```diff
@@ -158,6 +159,17 @@ func workloadScalingPolicyResourceSchema(function string, overhead float64) *sch
 				Description:      "The look back period in seconds for the recommendation.",
 				ValidateDiagFunc: validation.ToDiagFunc(validation.IntBetween(24*60*60, 7*24*60*60)),
 			},
+			"min": {
+				Type:        schema.TypeFloat,
```

Comment:
not sure if we have validation in api, but does this value can be less than the default? 🤔

Reply:
It can, and it should probably not. Will add validation to API 🙏